deploy/mayastor-daemonset.yaml

```diff
@@ -67,6 +67,8 @@ spec:
         volumeMounts:
         - name: device
           mountPath: /dev
+        - name: udev
+          mountPath: /run/udev
```

Comment:
Q: Do we need to mount as Read-Only?

Reply:
I believe that a readonly mount is most likely sufficient, but without completely thorough testing it is actually safer to leave as is.